reference/functions/pgroonga-highlight-html.md

```diff
@@ -300,9 +300,68 @@ SELECT
    from posts where title &@~ 'Search Words' or body &@~ 'Search Words' LIMIT 10 OFFSET 100;
 ```
 
+## Practical example 2: keyword search and highlight using with `NormalizerTable`
+
+If you specify `NormalizerTable`, like in this `CREATE INDEX USING pgroonga` document ([`How to use NormalizerTable` section][create-index-using-pgroonga]), the specified PGroonga index must have not only `TokenNgram` with `"report_source_location", true"` option but also both `Normalizer` and `NormalizerTable` with `"report_source_offset", true"` option for each.
+
+Here is an example (We use Japanese Name 'Saito' (`斉藤`) which has many variants):
+
+```sql
+CREATE TABLE memos (
+  content text
+);
+
+CREATE TABLE "synonyms" (
+   "target" text not null,
+   "normalized" text not null
+);
+
+INSERT INTO synonyms VALUES ('齊', '斉');
+INSERT INTO synonyms VALUES ('斎', '斉');
+INSERT INTO synonyms VALUES ('齋', '斉');
+
+INSERT INTO memos  (content)  VALUES ('斎藤さんの恋愛');
+INSERT INTO memos  (content)  VALUES ('斉藤さんの失恋');
+INSERT INTO memos  (content)  VALUES ('齋藤さんの片想い');
+
+CREATE INDEX pgroonga_synonyms_index ON synonyms
+ USING pgroonga (target pgroonga_text_term_search_ops_v2)
+                INCLUDE (normalized);
+
+CREATE INDEX pgroonga_memos_index
+          ON memos
+       USING pgroonga (content)
+        WITH (
+         tokenizer='TokenNgram(
+                          "unify_alphabet", false,
+                          "unify_symbol", false,
+                          "unify_digit", false,
+                          "report_source_location", true
+                        )',
+         normalizers='
+               NormalizerNFKC150("report_source_offset", true),
+               NormalizerTable(
+                  "normalized", "${table:pgroonga_synonyms_index}.normalized", 
+                  "target", "target",
+                  "report_source_offset", true)'
+            );
+```
+
+```sql
+select pgroonga_highlight_html(content, '{斉藤}', 'pgroonga_memos_index')
+                  from memos where content &@~ '斉藤';
+--             pgroonga_highlight_html            
+-- -----------------------------------------------
+--  <span class="keyword"></span>斎藤さんの恋愛
+--  <span class="keyword"></span>斉藤さんの失恋
+--  <span class="keyword"></span>齋藤さんの片想い
+-- (3 rows)
+```
+
 
 ## See also
 
   * [`pgroonga_query_extract_keywords` function][query-extract-keywords]
 
 [query-extract-keywords]:pgroonga-query-extract-keywords.html
+[create-index-using-pgroonga]:/reference/create-index-using-pgroonga.html#normalizer-table
```

Comment:
We need to use relative path to work with both of English and Japanese pages without additional translation:

```suggestion
[create-index-using-pgroonga-normalizer-table-highlight-html]:./create-index-using-pgroonga.html#normalizer-table-highlight-html
```

Reply:
Hmm, my markdown skill is very limited 😳
Thanks 👍